canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.scss

```diff
@@ -22,6 +22,19 @@
 		margin-bottom: $spacing-03;
 	}
 	.properties-checkboxset-container {
+		.checkbox-tooltip-container{
+			display: grid;
+			align-items: center;
+			grid-template-columns: 1fr 12fr;
+			width: 100%;
+		}
+		.tooltip-container{
```

Comment:
This is duplicate to 
```
.properties-label-container, .properties-summary-link-container {
	.tooltip-container {
		display: flex;
		width: 16px;
		height: 16px;
		margin-left: $spacing-03;
		cursor: pointer;
	}
}
```

We should be able to reuse that here and in radioset.  You can do that by setting another class on the tooltip, maybe `properties-icon-tooltip` then adding that to the list where this is set.

Reply:
Added the class in both.

Reply:
So we don't want the duplicate code.  You should be able to remove these css change and add `properties-icon-tooltip` to `.properties-label-container, .properties-summary-link-container`